README.md

```diff
@@ -25,7 +25,7 @@ structures:
 * [Bifunctor](#bifunctor)
 * [Profunctor](#profunctor)
```

Comment:
I'm totally with you. A program like `dot` which generates an ASCII diagram would be neat!

I very much like your change. Thanks for taking the time to work on it.